cognite_toolkit/_cdf_tk/commands/clean.py

```diff
@@ -275,25 +274,23 @@ def _remove_duplicates(
 
 class CleanCommand(CleanBaseCommand):
     def execute(
-        self, ctx: typer.Context, build_dir: str, build_env_name: str, dry_run: bool, include: list[str]
+        self, ctx: typer.Context, build_dir_raw: str, build_env_name: str, dry_run: bool, include: list[str]
```

Comment:
Hva menes med "raw"? At det er en str?

Reply:
yes, that is correct. With our current configuration of mypy it is not allowed to reuse a variable with another type.